pycd/settings.py

```diff
@@ -241,3 +241,11 @@
 STATIC_ROOT = os.path.join(os.path.dirname(os.path.abspath(__file__)),
                            os.environ.get('STATIC_ROOT', 'static'))
 STATICFILES_STORAGE = 'whitenoise.django.GzipManifestStaticFilesStorage'
+
+# amazon s3
```

Comment:
Amazon s3 interface, actually rados gw or openstack swift object storage interface can be used too